pegasus/forms/hoc_survey_2015.rb

```diff
@@ -0,0 +1,116 @@
+class HocSurvey2015
+
+  def self.normalize(data)
+    result = {}
+
+    result[:email_s] = Poste.decrypt(data[:code_s])
+    puts result[:email_s]
```

Comment:
No. It's leftover from last year: https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/forms/hoc_survey_2014.rb#L7

I'll remove.